src/pinecone.ts

```diff
@@ -157,35 +216,39 @@ export class Pinecone {
    * @param indexName - The name of the index to describe.
    * @returns A promise that resolves to {@link IndexMeta}
    */
-  describeIndex: ReturnType<typeof describeIndex>;
+  describeIndex(indexName: IndexName) {
+    return this._describeIndex(indexName);
+  }
 
   /**
    * List all Pinecone indexes
    * @example
    * ```js
-   * const indexes = await client.listIndexes()
+   * const indexes = await pinecone.listIndexes()
    * console.log(indexes)
    * // [ 'my-index', 'my-other-index' ]
    * ```
    *
    * @returns A promise that resolves to an array of index names
    */
-  listIndexes: ReturnType<typeof listIndexes>;
+  listIndexes() {
+    return this._listIndexes();
+  }
 
   /**
    * Creates a new index.
    *
    * @example
    * The minimum required configuration to create an index is the index name and dimension.
    * ```js
-   * await client.createIndex({ name: 'my-index', dimension: 128 })
+   * await pinecone.createIndex({ name: 'my-index', dimension: 128 })
    * ```
    * @example
    * In a more expansive example, you can specify the metric, number of pods, number of replicas, and pod type.
    * ```js
-   * await client.createIndex({
+   * await pinecone.createIndex({
```

Comment:
As a style thing, I'm trying to always reference `Pinecone` instances with a variable called `pinecone`.